test/integration/lerna-bootstrap.test.js

```diff
@@ -65,6 +67,22 @@ describe("lerna bootstrap", () => {
       const stdout = await execa.stdout(LERNA_BIN, ["run", "test", "--", "--silent"], { cwd });
       expect(stdout).toMatchSnapshot("--npm-client yarn: stdout");
     });
+
+    test.concurrent("passes remaining arguments to npm client", async () => {
+      const cwd = await initFixture("BootstrapCommand/npm-client-args");
```

Comment:
We also need an integration test that _doesn't_ employ the `npmClientArgs` key in lerna.json (using the `BootstrapCommand/basic` fixture, for example). I'm pretty sure that fails right now, as is.